app/models/user.rb

```diff
@@ -76,6 +76,13 @@ def accessible_groups
       )
     elsif role_type == TeamMemberRole::TYPE || role_type == SiteCoordinatorRole::TYPE
       VitaPartner.sites.where(id: role.site.id)
+    elsif role_type == CoalitionLeadRole::TYPE
+      coalition = Coalition.find(role.coalition.id)
+      organizations = coalition.organizations
+      sites = organizations.reduce([]) do |sites, organization|
+        [*sites, *organization.child_sites]
+      end
+      [coalition, *organizations, *sites]
```

Comment:
it seems to me there are approx a million ways to collect these things - is this the best? i don't know. i tried to think of a way that involved fewer queries but i could not

Reply:
which is not to say that i think this is a lot of queries, only that if there is a more optimal path i could not think of it

Reply:
this wouldn't affect the time to query but could also use a case statement